client/blocks/stripe-utils/type-defs.js

```diff
@@ -280,11 +280,9 @@
 /**
  * @typedef {Object} StripeServerData
  *
- * @property {string}                      stripeTotalLabel     The string used for payment
- *                                                              descriptor.
- * @property {string}                      publicKey            The public api key for stripe
+ * @property {string}                      key                  The public api key for stripe
  *                                                              requests.
- * @property {boolean}                     allowPrepaidCard     True means that prepaid cards
+ * @property {boolean}                     allow_prepaid_card   True means that prepaid cards
```

Comment:
I believe this is a `string` as well.

Reply:
Fixed in 677655fb32d043ba61f6cf46bbea39eb845b0699